src/test/storyshots.test.js

```diff
@@ -1,6 +1,33 @@
-import initStoryshots from '@storybook/addon-storyshots'
+import initStoryshots, { Stories2SnapsConverter } from '@storybook/addon-storyshots'
+import { mount } from '@vue/test-utils'
 
 initStoryshots({
   suite: 'RippleStoryshots',
-  configPath: './src/.storybook'
+  configPath: './src/.storybook',
+  // Below is the option that activates the async behaviour
+  // https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core#storyshots-for-async-rendered-components
+  asyncJest: true,
```

Comment:
This is to make storyshots to wait for async components(e.g vue carousel), followed instruction on https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core#storyshots-for-async-rendered-components 
However it turns one snapshot for each story instead of what we have before.